save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -65,8 +72,15 @@ class WarnPlugin(testConfig: TestConfig) : Plugin(testConfig) {
                 }
             }
             .mapValues { it.value.sortedBy { it.message } }
-        // todo: create a temp file with technical comments removed and feed it to the tool
-        val executionResult = pb.exec(warnPluginConfig.execCmd, null)
+
+        var execCmd: String = warnPluginConfig.execCmd
+
+        if (generalConfig.ignoreSaveComments) {
+            createTestFile(path)
+            execCmd = warnPluginConfig.execCmd + " test_file"
+        }
```

Comment:
```suggestion

        val execCmd = if (generalConfig.ignoreSaveComments) {
            createTestFile(path)
            warnPluginConfig.execCmd + " test_file"
        } else {
            warnPluginConfig.execCmd
        }
```
I don't like mutable properties :)